modules/inception/gcp/bastion-startup.tmpl

```diff
@@ -10,7 +10,7 @@ apt-add-repository "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https:/
 
 # Keep make and terraform the first items installed as they are needed
 # for testflight to complete
-apt-get update && apt-get install -y make terraform jq tree wget redis postgresql vault
+apt-get update && apt-get install -y make terraform jq tree wget redis postgresql vault google-cloud-sdk-gke-gcloud-auth-plugin
```

Comment:
This didn't work before, and I [removed it](https://github.com/GaloyMoney/galoy-infra/pull/77). Debugging it, I found that the `apt-add-repo` on line 9 wasn't working. Re-adding it without fixing the addition of the repo wouldn't work.

Reply:
changed the apt key and repo addition to the current recommendation at https://cloud.google.com/sdk/docs/install#deb (works on the bastions), but removed the `sudo`-s as it si running with root already

left the
```

apt-get install -y google-cloud-sdk-gke-gcloud-auth-plugin
```
line separate and last so nothing else will be missing if it fails.

Hope this will work.